tutorials/cyber/raw_data_generator/data_generator.yml

```diff
@@ -0,0 +1,10 @@
+name: data_generator
```

Comment:
see comment above about whether this additional environment can be eliminated in favor of adding these deps to the `rapids` env included in the container

Reply:
Completed